app/adapters/pii.ts

```
@@ -0,0 +1,33 @@
+    fileId?: string,
+    piiExtractionId?: string
+  ) {
+    const filesURL = `${this.namespace_v2}/pii_request/${piiExtractionId}/pii_data`;
+
+    return this.buildURLFromBase(filesURL);
+  }
+
+  urlForQuery(query: PiiDataQuery, modelName: string | number) {
+    const { fileId, piiExtractionId } = query;
+
+    delete query.fileId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)